### PR TITLE
Docs/readme

### PR DESCRIPTION
**Pull request by noctera**
This PR updates the Readme page of the documentation to make it look more appealing.

### CODE REVIEW

File: docs/home.md [hunk 1]
```diff
@@ -7,3 +25,37 @@
...
+operating system. Again, we have put a lot of emphasis on making it as platform-independent as possible. To find and
+download the right client for your operating system, take a look at our
+[desktop installation guide](vocascan-desktop/installation).
+
+?> We are currently working on a major architecture restructuring in order to be able to use Vocascan in the browser
+very soon.
+
+If any questions have come up, it's best to check out our [FAQ](general/faq). If they are still not answered, feel free
+to contact a maintainer. Check the [contact](#contact) section for more information.
+
```
Comment (luwol03) [inline docs/home.md:57]: can we also add a general roadmap here

File: .markdownlintignore [hunk 2]
```diff
@@ -1 +1,2 @@
 node_modules/
+docs/home.md
```
Comment (luwol03) [inline .markdownlintignore]: doesnt that work with a at the beginning of the file?
```html
<!-- markdownlint-disable MD041 MD033 -->
```

File: docs/home.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+    ·
+    <a href="https://github.com/vocascan/vocascan-desktop/discussions">Request Feature</a>
+    ·
+    <a href="https://discord.vocascan.com">Discord</a>
+  </p>
+</div>
+
 # Welcome to Vocascan
 
 > Digitalization is creating a huge change in the world. We no longer only have the chance to communicate with our
```
Comment (luwol03) [inline docs/home.md]: shouldnt we remove the `>` formatting?

File: .markdownlintignore (hunk 2 above)
Comment (noctera) [reply .markdownlintignore]: yeah this works

**Review by luwol03 — APPROVED** [review]

**Pull request merged**